.werft/jobs/build/prepare.ts

```diff
@@ -51,6 +52,7 @@ async function createVM(werft: Werft, config: JobConfig) {
         "TF_VAR_preview_name": config.previewEnvironment.destname,
         "TF_VAR_vm_cpu": `${cpu}`,
         "TF_VAR_vm_memory": `${memory}Gi`,
+        "TF_VAR_infra_provider": `${infra}`,
```

Comment:
Tiny nit: We can get rid of the string interpolation.

```suggestion
        "TF_VAR_infra_provider": infra,
```